### PR TITLE
Add eth_call to client + message typing

### LINKED ISSUE #74: Add eth_call support

### PR DESCRIPTION
**Pull request by KlonD90**
Close #74 

* Add eth_call
* Typing for message

### CODE REVIEW

File: src/types/ProcessedMessage.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import type { Address } from "abitype";
+import type { Hex } from "viem";
```
Comment (ukorvl) [inline src/types/ProcessedMessage.ts:2]: use our type

Comment (ukorvl) [reply src/types/ProcessedMessage.ts:2]: this is important because we will remove viem

**Review by ukorvl — COMMENTED** [review]
LGTM

File: src/types/ProcessedMessage.ts (hunk 1 above)
Comment (KlonD90) [reply src/types/ProcessedMessage.ts]: updated look please

**Review by ukorvl — APPROVED** [review]

**Pull request merged**